### PR TITLE
Add Multi Language Support To Page

### PR DESCRIPTION
**Pull request by pjanicked**
Moved Language from ModuleHotel to Public schema, Added PageTranslation class for language support to Pages
Issue: https://github.com/whynotearth/meredith-core/issues/154

### CODE REVIEW

File: WhyNotEarth.Meredith.Data.Entity/WhyNotEarth.Meredith.Data.Entity.csproj [hunk 1]
```diff
@@ -12,9 +17,10 @@
...
     </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.3" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Migrations" />
-  </ItemGroup>
```
Comment (ctyar) [inline WhyNotEarth.Meredith.Data.Entity/WhyNotEarth.Meredith.Data.Entity.csproj:19]: I'm not sure changes to `WhyNotEarth.Meredith.Data.Entity/WhyNotEarth.Meredith.Data.Entity.csproj` are necessary

File: WhyNotEarth.Meredith.Data.Entity/Migrations/20200423085701_AddMultiLanguageSupportForPage.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+namespace WhyNotEarth.Meredith.Data.Entity.Migrations
+{
+    public partial class AddMultiLanguageSupportForPage : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "CallToAction",
+                table: "Pages");
```
Comment (ctyar) [inline WhyNotEarth.Meredith.Data.Entity/Migrations/20200423085701_AddMultiLanguageSupportForPage.cs:12]: We have data in the `Page` table so dropping these columns will result in loss of data.
We need to move our current data which is in English to the translation table as part of the migration and then drop these columns.

Comment (pjanicked) [reply WhyNotEarth.Meredith.Data.Entity/Migrations/20200423085701_AddMultiLanguageSupportForPage.cs:12]: Yes agreed. How are we planning to do that - Entity framework migrations or manually making queries? Given the data - I can provide queries too. Either works

File: WhyNotEarth.Meredith.Data.Entity/WhyNotEarth.Meredith.Data.Entity.csproj (hunk 1 above)
Comment (pjanicked) [reply WhyNotEarth.Meredith.Data.Entity/WhyNotEarth.Meredith.Data.Entity.csproj:19]: The csproj changes were caused due to addition of EntityFrameworkCore.Tools package.

File: WhyNotEarth.Meredith.Data.Entity/Migrations/20200423085701_AddMultiLanguageSupportForPage.cs (hunk 2 above)
Comment (ctyar) [reply WhyNotEarth.Meredith.Data.Entity/Migrations/20200423085701_AddMultiLanguageSupportForPage.cs:12]: EF migration please

**Comment by ctyar** [comment]
Merged in https://github.com/whynotearth/meredith-core/commit/5e3fb61e4feb553f5c520bc8503ec352f4461450
Thank you @pjanicked

**Pull request closed without merging**